src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -491,12 +511,13 @@ def run(self):
             logger.info(f"Execute jobs taken from runtime")
             worker_cnt += self.__execute_tests(filters_runtime, worker_cnt)
         not_runned_test_filter, interapted_tests = self.__find_not_runned_tests()
-        if len(not_runned_test_filter) > 0:
-            logger.info(f"Execute not runned {len(not_runned_test_filter)} tests")
-            worker_cnt += self.__execute_tests(not_runned_test_filter, worker_cnt)
-        if len(interapted_tests) > 0:
-            logger.info(f"Execute interapted {len(interapted_tests)} tests")
-            worker_cnt += self.__execute_tests(interapted_tests, worker_cnt)
+        if (self._repeat_failed > 0) :
```

Comment:
MINOR:         `if (self._repeat_failed > 0) and len(not_runned_test_filter) > 0:`

Reply:
yes, and maybe it would be better to make this argument boolean until we need to repeat tests more then once

Reply:
For func tests is totally ok.
Default value should be 1 re-run